npm/private/test/parse_pnpm_lock_tests.bzl

```diff
@@ -44,30 +76,15 @@ def _parse_lockfile_v5_test_impl(ctx):
 }
 """)
 
+    # v5 formatted peer deps different in many ways including how peer packages were escaped
+    v5_expected_packages = dict(expected_packages)
```

Comment:
This highlights the differences between the "norm" and v5. I think these v5 vs v6 diffs is a bug and v5 is the one which should change, but that might not be worth our time at this point.

Note the v9 version of this are correct exceptions, data no longer in the lockfile (`requiresBuild` being the major one).